src/EA.Weee.Web/Areas/Producer/Views/Producer/ViewOrganisation/_OrganisationDetailsTabs.cshtml

```diff
@@ -20,21 +23,67 @@
     <h1 class="govuk-heading-l">@title</h1>
 </header>
 
+
+
+@if (hasYear)
+{
+    <div class="govuk-form-group @Html.Gds().FormGroupClass(m => m.Year)">
+        @Html.Gds().LabelFor(m => m.Year, new { @class = "govuk-label--s" })
```

Comment:
@seanciv To cover off users that dont have JS enabled you can wrap the button in a form and have a no script tag similar to C:\repos\prsd-weee\src\EA.Weee.Web\Views\Shared\_ReturnsFilter.cshtml that will display a button when JS is off. The form will just do the GET to the action that has been altered